lib/telegram-bot.js

```diff
@@ -1,8 +1,12 @@
 const _ = require('lodash');
 const Telegraf = require('telegraf');
+const extra = require('telegraf/extra');
+//const markup = extra.markdown();
+const markup = extra.HTML();
 const { config } = require('./config');
 const cache = {};
 let bot;
+let PausedChats = [];
```

Comment:
1. This is a separate feature.
2. Variable names should be camelCased instead of TitleCased.